components/api/constants/constants.bal

```diff
@@ -48,6 +49,7 @@ public const string IDP_DEFAULT_USERNAME = "admin";
 public const string AUTHORIZATION_HEADER = "Authorization";
 public const string COOKIE_HEADER = "Cookie";
 public const string TOKEN_COOKIE_KEY = "chpat";
+public const string TOKEN_BEARER_KEY = "Bearer";
```

Comment:
Shall we rename this constant to indicate that it is the header key ? Check the cookie header constant.